install/install-wal2json.sh

```diff
@@ -0,0 +1,34 @@
+echo "${_group}Downloading and installing wal2json ..."
+
+FILE_TO_USE="../postgres/wal2json/wal2json.so"
+ARCH=$(uname -m)
+FILE_NAME="wal2json-Linux-$ARCH.so"
+
+DOCKER_CURL="docker run --rm curlimages/curl"
+
+if [[ $WAL2JSON_VERSION == "latest" ]]; then
+    VERSION=$(
+        $DOCKER_CURL https://api.github.com/repos/getsentry/wal2json/releases/latest |
+        grep '"tag_name":' |
+        sed -E 's/.*"([^"]+)".*/\1/'
+    )
+
+    if [[ ! $VERSION ]]; then
+        echo "Cannot find wal2json latest version"
+        exit 1
+    fi
+else
+    VERSION=$WAL2JSON_VERSION
+fi
+
+mkdir -p ../postgres/wal2json
+if [ ! -f "../postgres/wal2json/$VERSION/$FILE_NAME" ]; then
+    mkdir -p "../postgres/wal2json/$VERSION"
+    $DOCKER_CURL -L \
+        "https://github.com/getsentry/wal2json/releases/download/$VERSION/$FILE_NAME" \
+        > "../postgres/wal2json/$VERSION/$FILE_NAME"
```

Comment:
I tried to mount a volume instead of redirecting the output, but I had permission issues in GHA (works fine locally). No matter how I did that, I did not manage to make the docker container write on a volume that mapped on any local directory. (I did not try to `chmod 777` a local directory, and I would rather not do that). 
- tried to mount the current directory
- tried to mount a directory in /tmp
- tried to set the --user option when running the container.
(These attempts with their error are all in the last few commits)
@BYK  did you ever try to do something like this?

Reply:
I don't really recall but I think using `stdout` is perfectly fine?